File: DSR Lecture notebooks/Module X - Practical Aspects - Patrick/Original scripts/-.m
base_path = "data/fraud-data/2017-01-"

rowlist = []
for i in range(1,32):
    filename = base_path + "{:02d}".format(i) + ".txt"
    with open(filename) as f:
        for line in f:
            rowlist.append( json.loads(line))
        
df=pd.DataFrame(rowlist)

File: DSR Lecture notebooks/Module 16 - Practical Aspects - Patrick/Original scripts/-.py
root_path = r"/home/batman/git/ref_DataScienceRetreat/DSR Lecture notebooks/Module X - Practical Aspects - Patrick/data/fraud-data"
files = []
dfs=list()
for fname in os.listdir(root_path):

    data = []
    with open(os.path.join(root_path,fname),'r') as f:
        for line in f:
            data.append(json.loads(line))
        
    files.append(data)
    dfs.append(pd.DataFrame(data))
